### ISSUE TITLE
blur password fields 

### ISSUE THREAD

**Issue by cantodorum** [MDeLuise/plant-it#399]
### Avoid duplicated feature requests

- [X] There are no open or closed feature requests that are related to this request

### Description

as of right now, the "password" fields on the "login" and the "change password" page don't "blur" the entered password(s). It is shown in clear text, which should be something like the common "dots"

e.g. portainer login screen
![image](https://github.com/user-attachments/assets/2d6feccd-7d05-4f20-a4fd-d121d72e5cc3)

this should prevent people from accidentally or intentionally seeing the password(s)

`App version`: 0.10.0
`Server version` 0.11.0

**Comment by MDeLuise** [comment]
Hi @cantodorum, thanks for opening the issue.

I want to clarify the current functionality: the password fields are designed to show the entered passwords in clear text by default, but there is a "hide" button at the end of each password field that allows users to toggle the visibility of their passwords if they choose to do so.

To better understand your concern, could you please elaborate on the specific issue you're experiencing? Are you suggesting that the passwords should be hidden by default, rather than requiring users to click the "hide" button?

I've attached some screenshots for reference:
<p align="center">
<img width="33%" alt="Screenshot 2025-01-14 at 11 24 21" src="https://github.com/user-attachments/assets/fb9fa760-5d3e-43b6-99e0-53b5cbb262a9" />
<img width="33%" alt="Screenshot 2025-01-14 at 11 24 26" src="https://github.com/user-attachments/assets/e4993bd8-d968-406a-af83-6476dabcb823" />
<img width="33%" alt="Screenshot 2025-01-14 at 11 25 16" src="https://github.com/user-attachments/assets/308b2c70-4fae-4d39-97b7-c89a378a0b52" />
</p>

**Comment by cantodorum** [comment]
Yes, I want the passwords to be blurred by default while maintaining the option to unblur them if you choose so, just as it is implemented now. This is the standard for most applications.

**Comment by MDeLuise** [comment]
Hi @cantodorum,
This task has been completed and will be implemented in the next project release.

Thank you for your contribution!
If you have any further questions or updates regarding this task, feel free to leave a comment below.
